turbopack/crates/turbopack-core/src/reference/mod.rs

```diff
@@ -230,6 +244,11 @@ impl ChunkableModuleReference for TracedModuleReference {
     fn chunking_type(&self) -> Vc<ChunkingTypeOption> {
         Vc::cell(Some(ChunkingType::Traced))
     }
+
+    #[turbo_tasks::function]
+    fn export_usage(&self) -> Vc<ExportUsage> {
+        ExportUsage::all()
+    }
```

Comment:
export_usage should not be implemented at all if it returns `ExportUsage::all()`. That's handled by the default implementation.